docs/pages/access-controls/idps/saml-attribute-mapping.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: SAML IdP Attribute Mapping
+description: How to map user attributes to custom SAML response
+h1: SAML Attribute Mapping
+---
+
+### Prerequisites
+
+(!docs/pages/includes/no-oss-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
```

Comment:
```suggestion
```
I think we can remove this since this isn't a how-to guide and we don't expect users to be able to quickly copy and paste commands.